merkledag/node.go

```diff
@@ -22,6 +22,16 @@ type ProtoNode struct {
 	encoded []byte
 
 	cached *cid.Cid
+
+	// Prefix specifies cid version and hashing function
+	Prefix cid.Prefix
+}
+
+var defaultCidPrefix = cid.Prefix{
+	Codec:    cid.DagProtobuf,
+	MhLength: -1,
```

Comment:
Should this really be -1?  Does it get set somewhere else.  Just double checking, so if you know this is correct than don't worry about it.